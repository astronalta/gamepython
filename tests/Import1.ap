import Import1, Import2 


ImportA < Object {

}
