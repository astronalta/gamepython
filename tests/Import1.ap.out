Module
  name: Import2
  feature0: Class
    name: ImportB
    comment:
  feature1: Import
    file: Import1.ap
    scope: Import1
Module
  name: Import1
  feature0: Class
    name: ImportA
    comment:
  feature1: Import
    file: Import1.ap
    scope: Import1
  feature2: Import
    file: Import2.ap
    scope: Import2
